### ISSUE TITLE
Japanese sentence with furigana can't be edited.

### ISSUE THREAD

**Issue by ckjpn** [Tatoeba/tatoeba2#1776]
I tried to remove the annotation from this sentence but couldn't.

私はいい先生（女）です。
https://tatoeba.org/eng/sentences/show/6860304

I assume that it has something to do with the manually edited furigana.

**Comment by trang** [comment]
After deploying 15809624141a97b765c9f496b1fdf81aedf00cb7 two weeks ago I edited the sentence `私はいい先生（女）です。`. I was then confused because the sentence didn't change but yet there is a log saying I modified the sentence.

It turns out that the sentence was properly edited. It is however still displayed as `私はいい先生（女）です。` because the transcription was not reset, due to the transcription feature not being migrated yet (#1761).

I'm closing this issue because Japanese sentences with furigana can now be edited. The remaining issue is that we have an inconsistency between the sentence and the transcription but that can be fixed when #1761 is fixed.

We should still think in general how to handle cases where a sentence is edited after it has been used as the source of other content (translations, transcriptions, audio...). 

For audio we handled this issue by disabling the possibility to edit. Only admins can bypass that, by temporarily setting the sentence to "no audio", then setting it back.
